client/src/components/ReviewsRatings/RatingsList.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import StarRating from './StarRating.jsx';
+import { ProgressBar } from 'react-bootstrap';
+
+const RatingsList = ({ reviews, reviewMetaData }) => {
+  const [averageRating, setAverageRating] = useState(0);
+  const [reviewRatings, setReviewRatings] = useState(0);
+  const [reviewChars, setReviewChars] = useState({});
+  console.log(reviewChars);
```

Comment:
Would you mind removing this console.log?

Reply:
Ah yes I removed it then added it again while I was editing my code a little. But yes its been commented out

Reply:
Would you mind just getting rid of it completely?